detector/detector.go

```diff
@@ -265,6 +255,31 @@ func DetectPkgCves(r *models.ScanResult, ovalCnf config.GovalDictConf, gostCnf c
 	return nil
 }
 
+// isPkgCvesDetactable checks whether CVEs is detactable with gost and oval from the result
+func isPkgCvesDetactable(r *models.ScanResult) bool {
+	if reuseScannedCves(r) {
+		logging.Log.Infof("Use CVEs as it as. Skip OVAL and gost detection")
+		return false
+	}
+
+	if r.Family == constant.ServerTypePseudo {
+		logging.Log.Infof("pseudo type. Skip OVAL and gost detection")
+		return false
+	}
+
+	if r.Release == "" {
+		logging.Log.Infof("r.Release is empty. detect as pseudo type. Skip OVAL and gost detection")
+		return false
+	}
+
+	if len(r.Packages)+len(r.SrcPackages) == 0 {
+		logging.Log.Infof("Number of packages is 0. Skip OVAL and gost detection")
+		return false
+	}
+
+	return true
+}
+
```

Comment:
Added function in order to clarify the conditions to be detected by OVAL and gost